### PR TITLE
Feat/basic structure

### PR DESCRIPTION
**Pull request by VladimirM89**
Implements: 
- add basic project structure
 - add eslint airbnb and prettier configs
 - add routing with lazy-loading modules

### CODE REVIEW

File: src/app/shared/shared.component.html
Comment (Ponomareva-Nina) [inline src/app/shared/shared.component.html:1]: no need

File: src/app/core/core.component.ts
Comment (Ponomareva-Nina) [inline src/app/core/core.component.ts:1]: no need of component in here. Instead of it - we need a module

**Review by Ponomareva-Nina — CHANGES_REQUESTED** [review]

File: src/app/shared/shared.component.ts
[the remark this answers is not in the document]
Comment (VladimirM89) [reply src/app/shared/shared.component.ts:1]: fix

File: src/app/shared/shared.component.html
Comment (VladimirM89) [reply src/app/shared/shared.component.html:1]: fix

File: src/app/core/core.component.ts
Comment (VladimirM89) [reply src/app/core/core.component.ts:1]: fix

File: src/app/core/core.component.html
[the remark this answers is not in the document]
Comment (VladimirM89) [reply src/app/core/core.component.html:1]: fix

**Review by avkomosko — APPROVED** [review]

**Review by Ponomareva-Nina — APPROVED** [review]
good job

**Review by Alextufeng — APPROVED** [review]

**Pull request merged**